packages/lodestar/src/chain/blocks/validate.ts

```diff
@@ -11,6 +11,10 @@ export function validateBlock(
   return (source) => {
     return (async function*() {
       for await(const job of source) {
+        if (job.reprocess) {
+          yield job;
+          continue;
+        }
```

Comment:
why?

Reply:
otherwise this condition will be failed: `computeEpochAtSlot(config, job.signedBlock.message.slot) <= finalizedCheckpoint.epoch`.
If we do want to reprocess I think we can ignore the checks.

Reply:
but let's say our last finalized epoch is 4, all non finalized blocks must have epoch >= 5 and won't trigger this check?

Reply:
you're right, this just happened during my development. I removed it in the latest version.